src/librustc_borrowck/borrowck/mod.rs

```diff
@@ -1299,9 +1299,10 @@ impl<'a, 'tcx> BorrowckCtxt<'a, 'tcx> {
                         snippet
                     );
                 } else {
-                    db.span_label(
+                    db.span_suggestion(
                         let_span,
-                        format!("consider changing this to `mut {}`", snippet),
+                        "consider changing this to",
+                        format!("mut {}", snippet),
                     );
```

Comment:
> I'm not super happy with the existing wording here, since it's now a suggestion.

What about "make this binding mutable":

```
LL |     let x: isize;
   |         - help: make this binding mutable: `mut x`
LL |     x = 1;
   |     ----- first assignment to `x`
```